projects/go-lib/src/lib/components/go-accordion/go-accordion-panel.component.ts

```diff
@@ -40,13 +42,18 @@ export class GoAccordionPanelComponent implements OnInit, OnChanges {
     this._expanded = expanded;
     this.containerClasses['go-accordion-panel--active'] = expanded;
     this.headerClasses['go-accordion-panel__header--active'] = expanded;
+    if (expanded) {
```

Comment:
I wonder if we should have some sort of `persistState` property that you could configure to refresh the component every time or not and default it to persist. Thought's on that?